gwtquery-core/src/test/java/com/google/gwt/query/client/GQueryCoreTestGwt.java

```diff
@@ -578,8 +578,8 @@ public void testPosition() {
   public void testPropMethod(){
     $(e).html("<input id=\"checkBox1\" type=\"checkbox\" checked=\"checked\" /> <input id=\"checkBox2\" type=\"checkbox\" />");
 
-    assertEquals(true, $("#checkBox1",e).prop("checked"));
-    assertEquals(false, $("#checkBox2",e).prop("checked"));
+    assertEquals(Boolean.TRUE, $("#checkBox1",e).prop("checked"));
```

Comment:
Basically Java 8 is more strict in generic typing and argument matching than Java 7, in this case it raises an error saying:
The method assertEquals(Object, Object) is ambiguous for the type GQueryCoreTestGwt